tests/tests.cpp

```diff
@@ -224,6 +224,27 @@ TEST_F(EngineBasicTest, TestBasicSortedOperations) {
     std::string t_skiplist(thread_skiplist + std::to_string(id));
     std::string k1, k2, v1, v2;
     std::string got_v1, got_v2;
+
+    if (id == 0) {
+      std::string key{""};
+      std::string value{"Empty key but with value"};
+      std::string sink{};
+      ASSERT_EQ(engine->SSet(overall_skiplist, key, value), Status::Ok);
+      ASSERT_EQ(engine->SGet(overall_skiplist, key, &sink), Status::Ok);
+      ASSERT_EQ(value, sink);
+      ASSERT_EQ(engine->SDelete(overall_skiplist, key), Status::Ok);
+    }
+
+    if (id == 0) {
```

Comment:
1. Every threads should write an empty key to thread_skiplist, the values should be different, then check get result.
2. After SDelete, check if SGet returns not found

Reply:
change overall_skiplist to global_skiplist, t_skiplist to thread_local_skiplist to avoid confusion.
add test for thread_local_skiplist for empty strings as key.